resources/lang/en/events/lead_ceremony_df.json

```
@@ -128,7 +128,572 @@
+        {
+          "text": "r_c's presence is like a cool wave of dark water, lapping at m_c's pawsteps. The dark spirit approaches with little fanfare and rests {PRONOUN/r_c/poss} muzzle on m_c's head to deliver a gift for [virtue]. {PRONOUN/r_c/subject/CAP} {VERB/r_c/murmur/murmurs} a word of advice, soft and cold as snowfall, then vanishes into the mist.",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunning", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+    "arrogant_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["arrogant"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "r_c approaches with a scornful gaze, already finding m_c lacking. Nevertheless, {PRONOUN/r_c/subject} {VERB/r_c/give/gives} a life for [virtue] and {VERB/r_c/warn/warns} m_c not to screw things up too much.",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunning", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+    "competitive_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["competitive"],
+      "rank": ["warrior", "deputy", "leader"],
+      "life_giving": [
+        {
+          "text": "r_c knows that m_c will never be able to outdo {PRONOUN/r_c/object}, even with the Dark Forest on the leader's side. Even as the life for [virtue] fades into a dull, painful ache, r_c whispers how {PRONOUN/r_c/subject} will be here and watching m_c fail again and again compared to dark spirit.",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+          "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunning", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+          "stubbornness", "authority"]
+        }
+      ]
+    },
+        "flamboyant_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["flamboyant"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "r_c approaches in a howl of cold wind, a shower of blood-red sparkles, and a flurry of shrieking bats. r_c whispers that {PRONOUN/r_c/subject} would've given m_c a life for making great entrances if {PRONOUN/r_c/subject} were allowed, but as it stands, m_c will have to make do with a life for [virtue].",
```

```suggestion
          "text": "r_c approaches in a howl of cold wind, a shower of blood-red sparkles, and a flurry of shrieking bats. r_c whispers that {PRONOUN/r_c/subject} would've given m_c a life for making great entrances if {PRONOUN/r_c/subject} were allowed to, but as it stands, m_c will have to make do with a life for [virtue].",
```
i'd die for flamboyant df cat.

~~add "rizz" to the virtues for this one i dare you~~